app.py

```diff
@@ -51,7 +56,9 @@ def hello_world():
     return jsonify({"message": "Hello, World!"})
 
 
+
 @app.route('/resume/experience', methods=['GET', 'POST', 'PUT'])
+@cross_origin()
```

Comment:
Can you add some documentation in the README about what this means. Other folks contributing may not be familiar